src/test/ui/rfc-1937-termination-trait/termination-trait-test-wrong-type.rs

```diff
@@ -4,5 +4,5 @@ use std::num::ParseIntError;
 
 #[test]
 fn can_parse_zero_as_f32() -> Result<f32, ParseIntError> { //~ ERROR
```

Comment:
This original error is also weird. it talks about `main`

Reply:
Filed https://github.com/rust-lang/rust/issues/60179